### DISCUSSION TITLE
allocated_draw should be related to device and not to PSU

### DISCUSSION THREAD

**Discussion by lucatortiglione** [netbox-community/netbox#12956] [General]
In a device i can insert a 800w or 1200w but power consumption will be always the same.

So Power Ports should inherit allocated_draw from device properties.

**Comment by pobradovic08** [comment]
Where exactly did you enter power consumption for device? I don't see a power consumption field for a device, just for PSUs.

**Reply by lucatortiglione** [reply]
Infact I think developers should fix this thing.

At the moment power consumption is just linked to PSU and not to the device
